docs/pages/reference/backends.mdx

```diff
@@ -581,6 +591,13 @@ To enable these options you will need to update the IAM Policy for Teleport.
 }
 ```
 
+<Admonition
+  type="warning"
+  title="pay_per_request with autoscaling"
+>
+  If `pay_per_request` is set to true, the settings for read/write capacity and auto scaling will be ignored.
```

Comment:
It's not a Boolean, so it can't be set to true. Please fix this.

Reply:
Looks like the [original PR](https://github.com/gravitational/teleport/pull/28672) has the same issue so @lxea please send a follow up to fix it in master as well.

Reply:
Yeah, sorry I missed the original. I approved with a few suggestions before these changes were made and a re-review was never requested.